app/services/action_service.rb

```diff
@@ -49,7 +49,7 @@ def remove_label(labels)
   end
 
   def assign_team(team_ids = [])
-    return unassign_team if team_ids[0].zero?
+    return unassign_team if team_ids[0]&.zero?
```

Comment:
Should we use `team_ids.empty?` instead?

Reply:
This is better. @vishnu-narayanan Please add a spec to cover the case.

Reply:
done

Reply:
@vishnu-narayanan I guess you missed @scmmishra's comment.